clocksilencer.ino

```diff
@@ -119,11 +128,14 @@ void synchroniseTime() {
     // if wifi found, break loop
   }
   Serial.println("WiFi connected");
-  //---------set with NTP---------------
-  configTime(3600, 3600, "europe.pool.ntp.org");
-  struct tm timeinfo;
+  //---------set with NTP--------------- 
+  // we have to rebuild this function to synchronise ext rtc
+  configTime(3600, 3600, "europe.pool.ntp.org");/
+  //==== //this need to be modified to meet the requirements of RTClib
+    struct tm timeinfo;//
   if (getLocalTime(&timeinfo)) {
```

Comment:
There might be some troubles after the time changes between summer / winter time.

But I think in such case it should be enough to disconnect from power and synchronize time from NTP...

Reply:
exactly.
Moreover, one must remember, this is a battery project, it needs battery charging once in few days ;)